activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -45,130 +45,780 @@ def primary_key(name, type = :primary_key, **options)
+        #
+        #   create_table :events, id: false do |t|
+        #     t.bigserial :id, primary_key: true
+        #   end
+        #
 
         ##
         # :method: bit
         # :call-seq: bit(*names, **options)
+        #
+        # Adds a +bit+ column for storing fixed-length bit strings.
+        #
+        # This type is useful for compactly representing flags or settings as binary values.
+        # By default, you can specify the number of bits the column should store.
+        #
+        # Example:
+        #
+        #   t.column :settings, "bit(8)"  # Stores exactly 8 bits
+        #
+        # When assigning values, use bit strings:
+        #
+        #   irb> user = User.new
+        #   user.settings = "01010101"  # This is a bit string
+        #   user.save!
 
         ##
         # :method: bit_varying
         # :call-seq: bit_varying(*names, **options)
+        #
+        # Adds a +bit_varying+ column for storing variable-length bit strings.
+        #
+        # This type is useful for representing binary flags, feature toggles, or compact
+        # settings as sequences of bits.
+        #
+        # Example:
+        #
+        #   t.bit_varying :settings
+        #
+        # You can also specify the column type explicitly:
+        #
+        #   t.column :flags, 'bit varying(16)'  # Up to 16 bits
+        #
+        # When assigning values, use bit strings:
+        #
+        #   irb> user = User.new
+        #   irb> user.flags = "01010101"  # Valid bit string
+        #   irb> user.save!
+        #
+        # If you assign an integer, it will be converted to a bit string:
+        #
+        #   irb> user = User.new
+        #   irb> user.flags = 42  # Stored as "00101010"
+        #   irb> user.save!
+        #
+        # If you assign a string that is not a valid bit string, it will raise an error:
+        #
+        #   irb> user = User.new
+        #   irb> user.flags = "invalid"  # Raises a PostgreSQL error
+
+        ##
+        # :method: box
+        # :call-seq: box(*names, **options)
+        #
+        # Adds a +box+ column for storing rectangular boxes in a 2D plane.
+        #
+        # The +box+ type is backed by PostgreSQL and represents a rectangle defined
+        # by two opposite corner points (typically the upper right and lower left).
+        #
+        # Example:
+        #
+        #   t.box :bounds
+        #
+        # Values can be assigned using PostgreSQL box syntax:
+        #
+        #   irb> Shape.create(bounds: '((1,1),(4,4))')
+        #
+        # This creates a rectangular box with opposite corners at (1,1) and (4,4).
+        #
+        # PostgreSQL treats the order of the points as irrelevant for storage, but
+        # typically the first point is considered the upper-right corner and the
+        # second the lower-left. The box spans the area between those two points on a 2D plane.
+
+        ##
+        # :method: bytea
```

Comment:
I think the method name should be `binary` here as well.
```suggestion
        # :method: binary
```